visualization/app/codeCharta/ui/codeMap/codeMap.render.service.ts

```diff
@@ -94,7 +94,7 @@ export class CodeMapRenderService implements OnDestroy {
     }
 
     getVisibleNodes(nodes: Node[]) {
-        return nodes.filter(node => node.visible && node.length > 0 && node.width > 0).sort((a, b) => b.height - a.height)
+        return nodes.filter(node => node.length > 0 && node.width > 0).sort((a, b) => b.height - a.height)
```

Comment:
i dont know why the part that checks if a node is visible was removed and if that is necessary but if so, we should also rename the function accordingly as it no longer only returns visible nodes